### DISCUSSION TITLE
Exception filter created by the exceptor doesn't work for `$replace` static filter

### DISCUSSION THREAD

**Discussion by uBlock-user** [uBlockOrigin/uBlock-issues#2952] [General] [answered]
STR --

- Add `||example.org^$replace=/Example//g` to my filters
- Open the logger and in another tab, navigate to `https://example.org/`
- Click on the rule and click on the exceptor widget to create a temp exception rule via the logger
- Refresh the page, observe that the rule is still applied(word "Example" is removed from the page) despite a temp exception filter visible in the logger.

Browser: Nightly

@gorhill

**Comment by gorhill** [comment, marked as answer]
https://github.com/gorhill/uBlock/commit/15e3f763a3f87a8f903f9462ad5164774b4f7d6a
